src/rules/rename-default-import.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * @fileoverview Rule to enforce aliases for default imports
+ * @author Michał Kołodziejski
+ */
+
+import docsUrl from '../docsUrl'
+
+
+function isDefaultImport(specifier) {
+  if (specifier.type === 'ImportDefaultSpecifier') {
+    return true
+  } else if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'default') {
+    return true
+  }
+  return false
+}
+
+function isCommonJSImport(declaration) {
+  const variableInit = declaration.init
+  if (variableInit.type === 'CallExpression') {
+    return variableInit.callee.name === 'require'
+  }
+  return false
+}
+
+function handleImport(context, node, specifierOrDeclaration, packageName, importAlias) {
+  const mappings = context.options[0] || {}
+
+  if (Object.keys(mappings).indexOf(packageName) === -1) {
+    return
+  }
+
+  if (mappings[packageName] !== importAlias) {
+    context.report({
+      node: node,
+      message: `Default import from '${packageName}' should be aliased to `
+              + `${mappings[packageName]}, not ${importAlias}`,
+      fix: fixImportOrRequire(specifierOrDeclaration, mappings[packageName]),
+    })
+
+    let declaredVariables
+    if (specifierOrDeclaration.type === 'VariableDeclarator') {
+      declaredVariables = context.getDeclaredVariables(specifierOrDeclaration.parent)[0]
+    } else {
+      declaredVariables = context.getDeclaredVariables(specifierOrDeclaration)[0]
+    }
+
+    for (const variableReference of declaredVariables.references) {
+      if (specifierOrDeclaration.type === 'VariableDeclarator' && variableReference.init) {
+        continue
+      }
+
+      context.report({
+        node: variableReference.identifier,
+        message: `Using incorrect binding name '${variableReference.identifier.name}' `
+                + `instead of ${mappings[packageName]} for `
+                + `default import from package ${packageName}`,
+        fix: fixer => {
+          if (variableReference.identifier.parent.type !== 'ExportSpecifier') {
```

Comment:
Am I misreading this or are you now simply skipping exports when fixing things? If a variable gets exported with the wrong name, then no occurrences of it nor its import should be renamed. Renaming it only in some places but not others would result in broken code.